generate/create_clf_data.py

```diff
@@ -75,44 +78,45 @@ def create_clf_data(dataset_name, dataset_path, image_name, image, data):
 
 def convert_data(dataset_type, num, offset=0):
 
-    new_dataset = 'clf_' + dataset_type
+    new_dataset = ('clf_' + dataset_type, ) * num # Broadcast our data to an num-len tuple for multithreading
     images_path = os.path.join(config.DATA_DIR, dataset_type, 'images')
-    new_images_path = os.path.join(config.DATA_DIR, new_dataset, 'images')
+    new_images_path = (os.path.join(config.DATA_DIR, new_dataset[0], 'images'), ) * num
 
-    os.makedirs(new_images_path, exist_ok=True)
+    os.makedirs(new_images_path[0], exist_ok=True)
```

Comment:
Will be an error for the same reason as line 83.

Reply:
Fixed